make.html

```diff
@@ -21,6 +22,17 @@ <h2>What is redirectatob?</h2>
       function update() {
         var u = 'https://redirectatob.github.io/index.html#';
         var r = document.getElementById('url').value;
+
+        // All modern browsers support the pattern attribute, but just in case:
+        if (r && !/^(https?:\/\/|data:)/i.test(r)) {
+          document.getElementById('error').innerText = 'You can only use http://, https://, and data:'
+          document.querySelector('a').innerText = ''
+
+          return 
+        } else {
+          document.getElementById('error').innerText = ''
```

Comment:
Since you have a `return` above, you may as well take this line out of the `else {}` block (which you can get rid of altogether).

Reply:
Semicolon style: after the `return`, or directly before `document`? Presumably the former since we decided semicolons were good style in this repo?

Reply:
After `return`.